RELEASE-NOTES.md

```diff
@@ -9,6 +9,10 @@
 
 ### New Features
 - The `CAR` distribution has been added to allow for use of conditional autoregressions which often are used in spatial and network models.
+- The dimensionality of model variables can now be parametrized through either of `shape`, `dims` or `size` (see [#4696](https://github.com/pymc-devs/pymc3/pull/4696)):
```

Comment:
A short doc with examples about these would be great to have in my opinion

Reply:
Where such a doc should live?
There are tons of other changes on `v4` that we should describe as well. Also the Readme section is essentially empty so far.
Let's write these docs after merging into master/main.

Reply:
For concrete examples about the shape/dims/size API please look at `test_shape_handling.py`

Reply:
Oh I didn't mean it should be a blocker at all!